### DISCUSSION TITLE
en/lc/3275/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#3485] [Announcements]
# en/lc/3275/

LeetCode solutions in any programming language

https://leetcode.doocs.org/en/lc/3275/

**Comment by DushyantTa** [comment]
Execute for input:
int[][] queries = {{7,7},{-9,4}}; 
int k = 1;
As per the competition, the program works,
In-case same code is executed in other machines, it will return output: 14,13 instead of -1 and 13.

**Reply by DushyantTa** [reply]
Language = Java
